### ISSUE TITLE
Package request: berkeleydb

### ISSUE THREAD

**Issue by bollwyvl** [conda-forge/staged-recipes#22320]
### Package name

berkeleydb

### Package version

Newest

### Package website

- https://www.jcea.es/programacion/pybsddb.htm

### Package availability

- https://pypi.org/project/berkeleydb/

### Additional comments

Appears to be pretty much @conda-forge/bsddb3 but with a name change, and some significant version differences.

This is a dependency for `rdflib[berkeleydb]` on modern pythons.

https://github.com/conda-forge/rdflib-feedstock/issues/25

### Package is not available

- [X] The package is not available on conda-forge.

### No previous issues or open PRs

- [X] No previous issue exists and no PR has been opened.

**Comment by jakirkham** [comment]
Would the two packages clobber each other? Do they need conflicts to avoid that?

**Comment by bollwyvl** [comment]
Yeah, it looks like they both import as the same name, so would need some `run_constrained`, but likely only on the new feedstock.

The old one won't even install on newer pythons, so i think the risk of clobberation is relatively small.
